scripts/fuzz_opt.py

```diff
@@ -87,6 +88,17 @@ def randomize_pass_debug():
     print('randomized pass debug:', os.environ.get('BINARYEN_PASS_DEBUG', ''))
 
 
+@contextlib.contextmanager
+def without_pass_debug():
```

Comment:
Maybe `no_pass_debug` would be a better name to make the `with` statement read more naturally?